src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -36,6 +36,24 @@ class PBSJobManager extends GridEngineBasedJobManager<PBSCommand> {
         super(executionService, parms)
     }
 
+    @Override
+    boolean checkCompatibilityToHost() {
+        // PBS tests are non trivial. qsub and co are not very helpful, as they don't feature any useful messages.
```

Comment:
Also note that some cluster managers (LSF, SGE) or even add-on packages (e.g. for Slurm) provide `qsub`-named wrappers for their CLI APIs. I cannot tell whether these are true `qsub` compatible. Similar for other commands that we need (`qstat`).